raiden-dapp/public/wizard.json.example

```diff
@@ -0,0 +1,4 @@
+{
+    "INFURA_ENDPOINT": "goerli.infura.io/v3/6d333faba41b4c3d8ae979417e281832",
```

Comment:
You need to include the schema, or at least `//` (to use the same schema as current page) in the beginning, or else it's going to be seen as a relative URL and fetched like `http://localhost:8080/goerli.infura.io/v3...`

Reply:
This is what is copied from Infura as endpoint, so with the highest usability. We don't fetch from `http://localhost:8080/goerli.infura.io/v3...`

Reply:
Then `https://` needs to be hardcoded. `fetch(url)` without schema treats it as relative.

Reply:
Oh, I noticed it's prepended on web3-provider. Thank you